pyproject.toml

```diff
@@ -92,6 +92,7 @@ extend-ignore = [
     "DTZ", # flake8-datetimez
     "TD", # flake8-todos
     "FIX", # flake8-fixme
+    "SIM", # flake8-simplify
```

Comment:
Ignore `SIM` rules. This wanted to single-line some if statements making them less readable.